automated install/basic-install.sh

```diff
@@ -1173,12 +1173,14 @@ setAdminFlag() {
             printf "  %b Web Interface Off\\n" "${INFO}"
             # or false
             INSTALL_WEB_INTERFACE=false
+            # Deselect the web server as well, since it is obsolete then
+            INSTALL_WEB_SERVER=false
             ;;
     esac
 
-    # Request user to install web server, if --disable-install-webserver has not been used (INSTALL_WEB_SERVER=true is default).
+    # Request user to install web server, if it has not been deselected before (INSTALL_WEB_SERVER=true is default).
     if [[ "${INSTALL_WEB_SERVER}" == true ]]; then
-        WebToggleCommand=(whiptail --separate-output --radiolist "Do you wish to install the web server (lighttpd)?\\n\\nNB: If you disable this, and, do not have an existing webserver installed, the web interface will not function." "${r}" "${c}" 6)
+        WebToggleCommand=(whiptail --separate-output --radiolist "Do you wish to install the web server (lighttpd) and required PHP modules?\\n\\nNB: If you disable this, and, do not have an existing webserver with required PHP modules installed, the web interface will not function.\nRequired PHP modules are: xml sqlite intl json\nAdditionally the web server needs to be member of the \"pihole\" group for full functionality." "${r}" "${c}" 6)
```

Comment:
Can we have the `PIHOLE_WEB_DEPS` be the source for the packages to get installed? Otherwise, when will almost certainly forget to update this bit when we have to add another dependency in, say, six months.

https://github.com/pi-hole/pi-hole/blob/d42785a3bfa793f9fa494096b27b6b7b72b0bbe3/automated%20install/basic-install.sh#L250

Reply:
@DL6ER 
Good idea:
```
$(for i in $PIHOLE_WEB_DEPS; do [[ $i == php* && $i != *-cgi ]] && echo -n "${i#*-} "; done)
```
- Not beautiful but we want the "php[X.Y]-" prefix and cgi not included, the latter since fpm or mod-php or other implementations would do as well.
- PHP json is missing on Debian then, since it is not in the list, most likely since it is dependency of all PHP handler packages, fpm, cgi and mod-php, hence pulled anyway. This is probably okay. We could add this one manually, but then needed to check for apt vs yum, otherwise it can be doubled. Or we could add it to the array on Ubuntu/Debian as well, also to be failsafe since it is definitely required?

Reply:
@DL6ER 
Okay done. I also shortened the output a bid so that it fits into a minimal whiptail:
![pihole](https://user-images.githubusercontent.com/28480705/81497857-91c26100-92c1-11ea-91fd-abd6d65d6554.png)